psi_cardinality/cpp/psi_cardinality_client_test_c.cpp

```diff
@@ -0,0 +1,141 @@
+//
+// Copyright 2020 the authors listed in CONTRIBUTORS.md
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "absl/strings/escaping.h"
+#include "absl/strings/str_cat.h"
+#include "bloom_filter_c.h"
+#include "crypto/ec_commutative_cipher.h"
+#include "gtest/gtest.h"
+#include "psi_cardinality_client_c.h"
+#include "rapidjson/document.h"
+#include "rapidjson/stringbuffer.h"
+#include "rapidjson/writer.h"
+#include "util/status_matchers.h"
+
+namespace psi_cardinality {
+namespace {
+
+class PSICardinalityClientBindingTest : public ::testing::Test {
```

Comment:
If we move the C bindings to their own directory, these tests should also get their own `cc_test` target.